package.json

```diff
@@ -42,8 +43,10 @@
     "eslint-plugin-jsx-a11y": "^6.0.3",
     "eslint-plugin-react": "^7.7.0",
     "fs": "0.0.1-security",
+    "http": "0.0.0",
```

Comment:
is http a needed dependency? I'm not familiar with this. it doesn't seem like it's being used. Also are you sure you need to have fs as a dependency? Node includes the fs module by default to my knowledge.